data-upgrade-users/src/main/resources/conf/portal/configuration.xml

```diff
@@ -74,7 +74,7 @@
           <name>plugin.upgrade.target.version</name>
           <description>The plugin target version (will not be executed if previous version is equal or higher than 6.3.1)
           </description>
-          <value>6.3.0</value>
+          <value>6.6.x</value>
```

Comment:
<value>6.6.x</value> -> <value>6.6.0</value> @sofyenne ?

Reply:
it is just to test acceptance, we set the correct version for the merge!